corebehrt/functional/utils.py

```diff
@@ -95,7 +126,14 @@ def filter_table_by_pids(df: pd.DataFrame, pids: List[str]) -> pd.DataFrame:
     Assumes that the table has a column named PID.
     Returns a new table with only the rows that have a PID in pids
     """
-    return df[df.PID.isin(pids)]
+    return df[df.PID.isin(set(pids))]
+
+
+def exclude_pids(data: dd.DataFrame, pids_to_exclude: List[str]) -> dd.DataFrame:
```

Comment:
Do we want to call a function that does nothing if the 2nd argument is empty? I get this is for performance reasons, but is it our task to optimize a dumb request?